doc/nrf/libraries/others/emds.rst

```diff
@@ -94,6 +94,40 @@ EMDS can store data within a guaranteed time, based on the amount of data being
 EMDS can be used to store data in memory in situations of critical power shortage, for example before the device battery is depleted.
 It is important that the hardware has the appropriate functionality to sustain power long enough for the storage to be completed before the power source is fully discharged.
 
+The easiest way of computing an estimate of the time required to store all entries, in a worst case scenario, is to call the :c:func:`emds_store_time_get` function.
+This function returns a worst-case storage time estimate in microseconds (µs) for a given application.
+For this to work, Kconfig options :kconfig:option:`CONFIG_EMDS_FLASH_TIME_BASE_OVERHEAD_US`, :kconfig:option:`CONFIG_EMDS_FLASH_TIME_ENTRY_OVERHEAD_US` and :kconfig:option:`CONFIG_EMDS_FLASH_TIME_WRITE_ONE_WORD_US` need to be set as described in the `Implementation`_ section.
+The :c:func:`emds_store_time_get` function estimates the required worst-case time to store :math:`n` entries using the following formula:
+
+.. math::
+
+   t_\text{store} = t_\text{base} + \sum_{i = 1}^n \left(t_\text{entry} + t_\text{word}\left(\left\lceil\frac{s_\text{ate}}{s_\text{block}}\right\rceil + \left\lceil\frac{s_i}{s_\text{block}}\right\rceil \right)\right)
+
+where :math:`t_\text{base}` is the value specified by :kconfig:option:`CONFIG_EMDS_FLASH_TIME_BASE_OVERHEAD_US`, :math:`t_\text{entry}` is the value specified by :kconfig:option:`CONFIG_EMDS_FLASH_TIME_ENTRY_OVERHEAD_US` and :math:`t_\text{word}` is the value specified by :kconfig:option:`CONFIG_EMDS_FLASH_TIME_WRITE_ONE_WORD_US`.
```

Comment:
Could you please also elaborated a bit more Kconfig help strings (or here) for `CONFIG_EMDS_FLASH_TIME_BASE_OVERHEAD_US` and `CONFIG_EMDS_FLASH_TIME_WRITE_ONE_WORD_US` and say that default values are taken from the datasheet for NVMC of the corresponding IC.

Reply:
Covered in the `Implementation` section.

Reply:
And to be honest I find it a bit weird that these even have "default" values? It is going to vary from case to case

Reply:
Yeah, true, they should depend on platform. I'm also not sure why they are configurable.